boards/FSM/Test/Src/Test_StateMachine.cpp

```diff
@@ -1,3 +1,4 @@
+#include <Test_FlowMeter.h>
```

Comment:
```suggestion

```

TestStateMachine can't really just use `Test_FlowMeter` because it requires two flow meters. I would just redefine `get_flow_rate` as `get_primary_flow_rate` and `get_secondary_flow_rate` in this file.